shell.nix

```diff
@@ -0,0 +1,13 @@
+{pkgs ? import <nixpkgs> {}, ...}:
+pkgs.mkShell {
+  buildInputs = with pkgs.buildPackages; [
+    pkg-config
+    rust-bin.stable.latest.default
+    llvmPackages_latest.bintools
```

Comment:
Why do we need this specifically? Is latest the right approach here? What does rust generally install when installed using rustup?